schema/artist/index.js

```diff
@@ -426,7 +426,7 @@ const ArtistType = new GraphQLObjectType({
         resolve: ({ id }, options) => {
           return gravity('related/sales', defaults(options, {
             artist_id: id,
-            sort: '-end_at',
+            sort: 'timely_at,name',
```

Comment:
So the `defaults` part of this is what sets up the override. If `sort` is passed in as a param, it will override this default setting.